model/trigger/process.go

```diff
@@ -37,7 +50,22 @@ func NotificationsFromEvent(e *event.EventLogEntry) ([]notification.Notification
 	catcher := grip.NewSimpleCatcher()
 	for i := range subscriptions {
 		n, err := h.Process(&subscriptions[i])
+		msg := message.Fields{
+			"source":              "events-processing",
+			"message":             "processing subscription",
+			"event_id":            e.ID,
+			"event_type":          e.EventType,
+			"event_resource_type": e.ResourceType,
+			"event_resource":      e.ResourceId,
+			"subscription_id":     subscriptions[i].ID,
+			"notification_is_nil": n == nil,
+		}
 		catcher.Add(err)
+		if err != nil {
+			grip.Error(message.WrapError(err, msg))
+		} else {
+			grip.Info(msg)
+		}
```

Comment:
I don't think you need to change this, but wrapped error messages (and errors themselves) are not loggable when they're nil, so you don't need to check if an err != nil before logging it, because grip will just noop the message. So you can do things like this as: 
```
grip.Error(message.WrapError(err, msg))
grip.InfoWhen(err == nil, msg)
```